docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_constructions.md

```diff
@@ -0,0 +1,88 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_constructions.md"]
+```
+
+# Constructions
+
+The standard way to define a polyhedron is by either giving a
+$V$-representation or an $H$-representation.  But polyhedra may also be
+constructed through other means: by name, via operations on other polyhedra, or
+from other objects in Oscar.
+
+## H and V representations
```

Comment:
```suggestion
## $H$- and $V$-representations
```

Reply:
Or just drop this intermediate headline? It feels a bit redundant. But of course that's a matter of taste, so feel free to keep it in.